app/controllers/api/v1/site_controller.rb

```diff
@@ -94,7 +97,9 @@ def view_study
               'Site'
           ]
           key :summary, 'Download a StudyFile'
-          key :description, 'Download a single StudyFile (via signed URL)'
+          key :description, "Download a single StudyFile (via signed URL)<br/><br/><strong>NOTE</strong>: Due to CORS issues, files cannot be " + \
+                            "downloaded via Swagger.  To download a file, either use a client such as Postman, or copy/paste " + \
+                            "the CURL command into a terminal and add the '-L' flag.".html_safe
```

Comment:
This works, assuming we can't hide the "Request URL" nor add the curl `-L` flag.  But a refinement for the note would help:

```suggestion
                            "the CURL command into a terminal and add the '-L' flag immediately before the URL.".html_safe
```

Reply:
I tried to figure out how to add in the `-L` flag, but it seems that the curl code is auto-generated in the Swagger JS, so we'd have to monkey-patch it, and only for this method.  That didn't seem like it would be very feasible.

Reply:
I see -- agreed!